src/App.js

```diff
@@ -1163,6 +1175,15 @@ export class App extends React.Component<AppProps, AppState> {
                                     <FormattedMessage id='App.appHeading'/>
                                 </a>
                             </h1>
+                            <div className='App__PrivacyModal__container'>
```

Comment:
I think something like `App__PrivacyButtonContainer` would be a better class name for this div: it's a container for the button, rather than for the modal.

Reply:
Done.